website/client/components/chat/chatCard.vue

```diff
@@ -222,6 +222,7 @@ export default {
   },
   methods: {
     async like () {
+      console.log(this);
```

Comment:
could you remove the `console.log` ?

Reply:
Removed (squashed my commits so the original doesn’t have this mistake).